db/schema.rb

```diff
@@ -1,25 +1,29 @@
-# This file is auto-generated from the current state of the database. Instead
-# of editing this file, please use the migrations feature of Active Record to
-# incrementally modify your database, and then regenerate this schema definition.
-#
-# Note that this schema.rb definition is the authoritative source for your
-# database schema. If you need to create the application database on another
-# system, you should be using db:schema:load, not running all the migrations
-# from scratch. The latter is a flawed and unsustainable approach (the more migrations
-# you'll amass, the slower it'll run and the greater likelihood for issues).
-#
-# It's strongly recommended that you check this file into your version control system.
-
-ActiveRecord::Schema.define(version: 20170127063628) do
-
-  create_table "users", force: :cascade do |t|

```

Comment:
The `users` table is actually here, it's just that the diff is messed up (see the added section below). While I was rebasing, I remember seeing some Windows line endings hanging around for some reason; it could be that Matthew generated this file at some point and rails on Windows likes using Windows line endings.

(If we wanted to make sure this doesn't happen in the future, we could add a pre-commit hook checking line endings, or make CI check for this. But there might be false positives.)